.travis.yml

```diff
@@ -1,25 +1,24 @@
-language: erlang
+language: bash
 sudo: false
 
-matrix:
-  include:
-    - os: linux
-      otp_release: 19.0
-    - os: linux
-      otp_release: 19.1
-    - os: linux
-      otp_release: 19.2
-    - os: linux
-      otp_release: 19.3
-    - os: linux
-      otp_release: 20.0
-    - os: linux
-      otp_release: 20.1
-    - os: linux
-      otp_release: 20.2
-
 env:
-  - ELIXIR_ASSERT_TIMEOUT=2000
+  - OTP_RELEASE=OTP-19.0 ELIXIR_ASSERT_TIMEOUT=2000
+  - OTP_RELEASE=OTP-19.1 ELIXIR_ASSERT_TIMEOUT=2000
+  - OTP_RELEASE=OTP-19.2 ELIXIR_ASSERT_TIMEOUT=2000
+  - OTP_RELEASE=OTP-19.3 ELIXIR_ASSERT_TIMEOUT=2000
+  - OTP_RELEASE=OTP-20.0 ELIXIR_ASSERT_TIMEOUT=2000
+  - OTP_RELEASE=OTP-20.1 ELIXIR_ASSERT_TIMEOUT=2000
+  - OTP_RELEASE=OTP-20.2 ELIXIR_ASSERT_TIMEOUT=2000
+  - OTP_RELEASE=OTP-20.3 ELIXIR_ASSERT_TIMEOUT=2000
+  - OTP_RELEASE=maint ELIXIR_ASSERT_TIMEOUT=2000
+  - OTP_RELEASE=master ELIXIR_ASSERT_TIMEOUT=2000
```

Comment:
https://docs.travis-ci.com/user/environment-variables/#Global-Variables

It seems it would be better to do:

```yaml
env:
  global:
    - ELIXIR_ASSERT_TIMEOUT=2000
  matrix:
    - OTP_RELEASE=OTP-19.0
    - ...
```